core/object.cpp

```diff
@@ -810,11 +809,7 @@ bool Object::has_method(const StringName &p_method) const {
 
 	MethodBind *method = ClassDB::get_method(get_class_name(), p_method);
 
-	if (method) {
-		return true;
-	}
-
-	return false;
+	return method;
```

Comment:
That doesn't look good. It's ok to do an implicit null check with `if (method)`, but not an implicit null check casted to boolean as return value IMO. So this should be `method != NULL`.